app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/dashboard/view/MyDashboardFragment.kt

```diff
@@ -32,6 +34,9 @@ import timber.log.Timber
  */
 class MyDashboardFragment : Fragment(), MyDashboardContract.View {
 
+    @BindView (R.id.rv_course_list)
+   lateinit var recy : RecyclerView
```

Comment:
Use https://kotlinlang.org/docs/tutorials/android-plugin.html#view-binding instead of butterknife, findviewbyid.

Reply:
still working on this aspect, forgot to reverse before I pushed